encoding/message.go

```diff
@@ -4,6 +4,14 @@
 // By Rene Nayman
 package encoding
 
+type Protocol string
+
+const (
+	RPC = "RPC"
+	GOSSIP = "GOSSIP"
+	PING  = "PING"
+)
+
 // Message represents a parsed Hobbits message.
 // See examples of unparsed and parsed messages here: https://github.com/deltap2p/hobbits/blob/master/specs/protocol.md
 type Message struct {
```

Comment:
you probably wanna change message protocol to be of the type `Protocol` rather than string now.